virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc

```diff
@@ -0,0 +1,15 @@
+[id="virt-attaching-vm-to-sriov-network"]
+= Attaching a virtual machine to an SR-IOV network
+include::modules/virt-document-attributes.adoc[]
+:context: virt-attaching-vm-to-sriov-network
+toc::[]
+
+You can attach a virtual machine to use a Single Root I/O Virtualization (SR-IOV) network as a secondary network.
+
+.Prerequisites
+
+* You must have xref:../../../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[configured an SR-IOV device].
+
+* You must have xref:../../../virt/virtual_machines/vm_networking/virt-defining-an-sriov-network.adoc#virt-defining-an-sriov-network[defined an SR-IOV network].
+
+include::modules/virt-attaching-vm-to-sriov-network.adoc[leveloffset=+1]
```

Comment:
Are there next steps for this assembly?

Reply:
From here there is just using the virtual machine that is now using the sriov network.